### PR TITLE
Get filesystem stats from influxdb

### LINKED ISSUE #1643: Update fs / fs detail pages to use influx as stats datasource
Update the following areas on the fs / fs detail pages to query directly from influx instead of relying on stale api data:

Screen Shot 2020-03-09 at 1 52 46 PM

Screen Shot 2020-03-09 at 1 53 19 PM

### PR DESCRIPTION
**Pull request by ip1981**
Closes #1643 

Signed-off-by: Igor Pashev <pashev.igor@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/1694)

### CODE REVIEW

File: iml-gui/crate/src/page/filesystem.rs [hunk 1]
```diff
@@ -6,33 +6,100 @@ use crate::{
...
+        let query = format!(
+            r#"SELECT SUM(b_total), SUM(b_free)
+                    , SUM(f_total), SUM(f_free)
+                    , SUM(clients)
+               FROM (SELECT LAST(bytes_total) AS b_total
+                          , LAST(bytes_free) AS b_free
+                          , LAST(files_total) AS f_total
+                          , LAST(files_free) AS f_free
+                      FROM target
+                      WHERE "kind" = '"OST"' AND "fs" = '"{fs_name}"' AND time > now() - 2m
```
Comment (ip1981) [inline iml-gui/crate/src/page/filesystem.rs]: `time > ...` is an attempt for https://github.com/whamcloud/integrated-manager-for-lustre/issues/1692

Comment (jgrund) [reply iml-gui/crate/src/page/filesystem.rs]: Can we leave this off for now?

File: iml-gui/crate/src/page/filesystems.rs [hunk 2]
```diff
@@ -4,41 +4,126 @@ use crate::{
...
 use seed::{prelude::*, *};
+use std::time::Duration;
 use std::{collections::HashMap, sync::Arc};
 
+const INFLUX_QUERY: &str = r#"
+SELECT SUM(b_total), SUM(b_free), SUM(clients)
+FROM (SELECT LAST(bytes_total) AS b_total
+           , LAST(bytes_free) AS b_free
+      FROM target
+      WHERE "kind" = '"OST"' AND time > now() - 2m
```
Comment (jgrund) [inline iml-gui/crate/src/page/filesystems.rs]: let's keep the time subtraction off for now.

File: iml-gui/crate/src/page/filesystems.rs [hunk 3]
```diff
@@ -4,41 +4,126 @@ use crate::{
...
+      GROUP BY target)
+   , (SELECT LAST(connected_clients) AS clients
+      FROM target
+      WHERE kind='"MDT"' AND time > now() - 2m
```
Comment (jgrund) [inline iml-gui/crate/src/page/filesystems.rs]: let's keep the time subtraction off for now.

File: iml-gui/crate/src/page/filesystems.rs [hunk 4]
```diff
@@ -134,6 +220,7 @@ pub fn view(cache: &ArcCache, model: &Model, all_locks: &Locks, session: Option<
                     .map(|f| match model.rows.get(&f.id) {
                         None => empty![],
                         Some(row) => {
+                            let stats = model.stats.get(&f.name).cloned().unwrap_or_default();
```
Comment (ip1981) [inline iml-gui/crate/src/page/filesystems.rs]: Could not find how to avoid cloning here.

**Pull request merged**